Sources/Alchemy/SQL/QueryBuilder/Query.swift

```diff
@@ -4,7 +4,7 @@ import NIO
 public class Query: Sequelizable {
     let database: Database
     
-    private(set) var columns: [SQL] = []
+    private(set) var columns: [SQL] = [SQL("*")]
```

Comment:
@pushchris `SELECT` didn't have a column default if `.select` wasn't called in the query builder chain; explicitly calling `select` overwrites `Query.column` anyway, so there doesn't seem to be an issue defaulting to this.